ui/app/components/policy-form.hbs

```diff
@@ -46,12 +46,12 @@
                 id="fileUploadToggle"
                 @type="checkbox"
                 name="fileUploadToggle"
-                class="switch is-rounded is-success is-small"
```

Comment:
Deleted `switch` css to consistently use `toggle`
### before + after
<img width="618" alt="Screenshot 2023-11-03 at 1 23 33 PM" src="https://github.com/hashicorp/vault/assets/68122737/42e6ab8f-295a-44e7-91c9-c993cc0873f8">